tests/atest/utils/__init__.py

```diff
@@ -118,14 +118,14 @@ def check_rule(
         if actual != expected:
             missing_expected = sorted(set(actual) - set(expected))
             missing_actual = sorted(set(expected) - set(actual))
-            present_in_actual = "\n    ".join(missing_expected)
-            present_in_expected = "\n    ".join(missing_actual)
-            raise AssertionError(
-                "Actual issues are different than expected.\n"
-                f"Actual issues not found in expected:\n    {present_in_actual}"
-                f"\n\nExpected issues not found in actual:\n    {present_in_expected}"
-            )
-        assert actual == expected, f"{actual} != {expected}"
+            error = "Actual issues are different than expected.\n"
```

Comment:
This change improves readability of errors in atests - it will not display full trace now, just the difference between actual and expected issues.